### PR TITLE
Minor upgrades

### PR DESCRIPTION
**Pull request by mbugni**

### CODE REVIEW

File: com.moddb.TotalChaosRetro.png
Comment (AsciiWolf) [inline com.moddb.TotalChaosRetro.png:1]: Any chance you could give me the right commands to modify the desktop icon for the non-Retro Total Chaos as well? :-) Thanks!

**Comment by AsciiWolf** [comment]
bot, build

**Comment by AsciiWolf** [comment]
I did only some very brief testing since I am overwhelmed with other work at the moment. The games seems to run fine, however I have noticed a strange flickering instead of the correct effect on the switches:

flickering

(However, it may be a Mesa bug since I was testing it on my laptop with a low-end Intel GPU.)

Also, the monster that should be present in this level was missing for some reason.

File: com.moddb.TotalChaosRetro.png
Comment (mbugni) [reply com.moddb.TotalChaosRetro.png:1]: It's [here](https://github.com/flathub/com.moddb.TotalChaosRetro/pull/1#issuecomment-1575105277). Adjust values. Last time I used 30 for remove more darkness, if I well remember.

**Comment by mbugni** [comment]
> I did only some very brief testing since I am overwhelmed with other work at the moment. The games seems to run fine, however I have noticed a strange flickering instead of the correct effect on the switches:
> 
> (However, it may be a Mesa bug since I was testing it on my laptop with a low-end Intel GPU.)
>

Try Display Options > Dynamic Light Options > Lights affect sprites > No
 
> Also, the monster that should be present in this level was missing for some reason.

:-| it's present for me, it appears after you pull the first lever.

**Comment by AsciiWolf** [comment]
> it appears after you pull the first lever.

Ah, that explains it. :-)

File: com.moddb.TotalChaosRetro.png
Comment (AsciiWolf) [reply com.moddb.TotalChaosRetro.png:1]: Could you provide me the updated command with adjusted values? I want the regular Total Chaos logo to have exactly the same changes as the Retro Edition one. Thanks!

**Pull request merged**